pkg/cpi/commonUtils.go

```diff
@@ -172,8 +172,9 @@ func (httpFileUploadRequestParameters HttpFileUploadRequestParameters) HandleHTT
 	if response == nil {
 		return errors.Errorf("did not retrieve a HTTP response: %v", httpErr)
 	}
+	responseCode := response.StatusCode
 
-	if response.StatusCode == http.StatusOK {
+	if (responseCode == http.StatusOK) || (responseCode == http.StatusCreated) {
```

Comment:
What I mean [here](https://github.com/SAP/jenkins-library/pull/3663#discussion_r845039081) is that you could make the **expected** response code (on the left of the comparisons) a part of the `HttpFileUploadRequestParameters`. Then you don't need to continuously adapt this if-clause when there is another expected response code.

Reply:
this is called by many commands, we need change all those commands to accommodate this. since in upload scenarios its always 202 or 201 is the case, we can live with this.

Reply:
ok